packages/mobile/src/home/WalletHome.tsx

```diff
@@ -116,6 +128,16 @@ export class WalletHome extends React.Component<Props, State> {
     // Waiting 1/2 sec before triggering to allow
     // rest of feed to load unencumbered
     setTimeout(this.tryImportContacts, 500)
+
+    this.setState({
+      isAccountBalanceZero: !ALL_CURRENCIES.some((currency) =>
```

Comment:
This works but imo it would be preferable to iterate through the values on `this.props.balances` instead of creating a new `ALL_CURRENCIES` array. The benefit to that approach is that it doesn't require the creation of an additional variable, which we'd need to remember to update. If the information we need is in `this.props.balances`, then we should only rely on that.

In this case, it would look like this:

```
    const hasBalance = Object.values(this.props.balances).some((balance) =>
      balance?.isGreaterThan(0)
    )
 ```